stix2/equivalence/object/__init__.py

```diff
@@ -9,30 +11,73 @@
 logger = logging.getLogger(__name__)
 
 
-def semantically_equivalent(obj1, obj2, prop_scores={}, **weight_dict):
-    """This method verifies if two objects of the same type are
-    semantically equivalent.
+def object_equivalence(obj1, obj2, prop_scores={}, threshold=70, **weight_dict):
+    """This method returns a true/false value if two objects are semantically equivalent.
+    Internally, it calls the object_similarity function and compares it against the given
+    threshold value.
 
     Args:
         obj1: A stix2 object instance
         obj2: A stix2 object instance
         prop_scores: A dictionary that can hold individual property scores,
             weights, contributing score, matching score and sum of weights.
+        threshold: A numerical value between 0 and 100 to determine the minimum
+            score to result in successfully calling both objects equivalent. This
+            value can be tuned.
         weight_dict: A dictionary that can be used to override settings
             in the semantic equivalence process
 
     Returns:
-        float: A number between 0.0 and 100.0 as a measurement of equivalence.
+        bool: True if the result of the object similarity is greater than or equal to
+            the threshold value. False otherwise.
 
     Warning:
-        Object types need to have property weights defined for the equivalence process.
+        Object types need to have property weights defined for the similarity process.
         Otherwise, those objects will not influence the final score. The WEIGHTS
         dictionary under `stix2.equivalence.object` can give you an idea on how to add
         new entries and pass them via the `weight_dict` argument. Similarly, the values
         or methods can be fine tuned for a particular use case.
 
     Note:
-        Default weights_dict:
+        Default weight_dict:
+
+        .. include:: ../../object_default_sem_eq_weights.rst
+
+    Note:
+        This implementation follows the Semantic Equivalence Committee Note.
+        see `the Committee Note <link here>`__.
+
+    """
+    similarity_result = object_similarity(obj1, obj2, prop_scores, **weight_dict)
+    if similarity_result >= threshold:
+        return True
+    return False
+
+
+def object_similarity(obj1, obj2, prop_scores={}, **weight_dict):
```

Comment:
A fewof the log messages in this function say "semantic equivalence" where "similarity" would be more accurate. Other places we can make the swap are the docstring for `_versioned_checks()` on line 353, the docstring for `reference_check()` on line 376, the docstring for `list_reference_check()` on line 397.

Reply:
done